pipeline-modules/app-service/aws/aws-scan.yaml

```diff
@@ -0,0 +1,88 @@
+provisioner: aws
+name: aws-container-scan
```

Comment:
this should be ecr-image-scan? since aws is redundant.
also we should have same file name so that it's easier to find the module.
Display name should also be change according to this.